Formula/xcbuild.rb

```diff
@@ -0,0 +1,24 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git",
+    :tag => "0.1.0",
+    :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa",
+    :shallow => false
+  head "https://github.com/facebook/xcbuild.git",
+    :shallow => false
```

Comment:
Yeah. We used to have a full example formula with detailed line references for people to glance through, but we scrapped it in favour of better split documentation & more real-world formulae usage examples. Sometimes I wonder if it would have still be useful 😓.